### PR TITLE
Not working

### PR DESCRIPTION
**Pull request by ayushdevelop**
# Related Issue

- https://github.com/code-mancers/g-bot/issues/5

# Functionalities Added or Changed

- On clicking the `Not Working today button` received after running the `standup` command the response with the name of user should be send to selected channel.

# Checklist

- [x] Uploaded screenshots

# Screenshots

Not Working Response

### CODE REVIEW

File: server/src/listeners/actions/notWorking.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      token: context.botToken,
+      // Channel to send message to
+      channel: process.env.GATHER_BOT_CHANNEL,
+      text: `*${user.display_name}* has marked as not working today`,
+    });
+  } catch (error) {
+    await client.chat.postMessage({
+      token: context.botToken,
+      // Channel to send message to
+      channel: "C0403497QV8",
```
Comment (SujayPrabhu96) [inline server/src/listeners/actions/notWorking.ts:51]: Move to env

File: server/src/index.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
-require("dotenv").config();
+require('dotenv').config();
 
-const { handleListeners } = require("./listeners");
-const { App, ExpressReceiver } = require("@slack/bolt");
+const { handleListeners } = require('./listeners');
+const { App, ExpressReceiver } = require('@slack/bolt');
 
-const express = require("express");
+const express = require('express');
```
Comment (SujayPrabhu96) [inline server/src/index.ts]: We haven't set `singleQuote` rule in prettier. Can you check why its included ?

File: server/src/listeners/actions/index.ts [hunk 3]
```diff
@@ -1,9 +1,11 @@
-const postStandup = require("./postStandup");
+const postStandup = require('./postStandup');
```
Comment (SujayPrabhu96) [inline server/src/listeners/actions/index.ts]: Add comment what `/listeners/actions` folder is for ? Similarly for `/listeners/commands` & `/listeners/views`

File: server/src/listeners/actions/notWorking.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}: NotWorkingArgs): Promise<void> => {
+  await ack();
+
+  /**
+   * Making a request to fetch user details
+   * to submit the response in the channel selected
+   * as user
+   */
+
+  const user = await client.users.profile
```
Comment (SujayPrabhu96) [inline server/src/listeners/actions/notWorking.ts:29]: Same method is used in `/views/modalViews`. Can you move it to `/utils` folder ?

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/src/utils/userDetails.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ * as user
+ */
+
+type UserDetailArgs = {
+  client: { users: { profile: { get: Function } }; token: string };
+  body: { user: { id: string } };
+};
+
+const userDetails = async ({ client, body }: UserDetailArgs) => {
+  const data = await client.users.profile
```
Comment (SujayPrabhu96) [inline server/src/utils/userDetails.ts:13]: Use ayanc/await here. That looks neat & avoid any. instead define type for the data
```
const data = await client.users.profile.get({ token: client, user: body.user.id });
const { profile: image_192, display_name } = data
```

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/src/utils/userDetails.ts (hunk 5 above)
Comment (ayushdevelop) [reply server/src/utils/userDetails.ts:13]: okay, will be taking care of it

**Review by SujayPrabhu96 — APPROVED** [review]

**Pull request merged**